src/components/modals/VideoModal.jsx

```
@@ -0,0 +1,202 @@
+                Exercise Tutorial
+              </h3>
+              <p className="text-zinc-400 text-sm">
+                {exercise?.name || "Exercise Video"}
+              </p>
+            </div>
+          </div>
+          <button
+            onClick={onClose}
+            className="p-2 hover:bg-zinc-800 rounded-lg transition-colors group"
+            title="Close video (Esc)"
+          >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)